doc/conf.py

```diff
@@ -266,42 +269,73 @@ def _check_dependencies():
 # The style sheet to use for HTML and HTML Help pages. A file of that name
 # must exist either in Sphinx' static/ path, or in one of the custom paths
 # given in html_static_path.
-html_style = f'mpl.css?{SHA}'
+# html_style = 'matplotlib.css'
+# html_style = f"mpl.css?{SHA}"
+html_css_files = [
+    f"mpl.css?{SHA}",
+]
+
+html_theme = "pydata_sphinx_theme"
 
 # The name for this set of Sphinx documents.  If None, it defaults to
 # "<project> v<release> documentation".
 # html_title = None
 
 # The name of an image file (within the static path) to place at the top of
 # the sidebar.
-# html_logo = 'logo.png'
+html_logo = "_static/logo2.svg"
+html_theme_options = {
+    "logo_link": "/stable/index.html",
```

Comment:
Is that what we want on versioned pages?

Reply:
Probably not.  This also makes the locally built version of the docs not self-contained (so does the third party page link, but that includes the full URL so it atleast works (assuming you have network).  I'll change this.